lib/models/assignment.rb

```diff
@@ -2,6 +2,14 @@ class Assignment < Forgery
   attr_reader :name, :description, :due_at, :lock_at, :course_id, :assignment_group
```

Comment:
i would probably use attr_accessor for due_at and lock_at

Reply:
resolved